src/protocols/device_path.rs

```diff
@@ -64,3 +64,19 @@ pub struct HardDriveMedia {
     pub partition_format: u8,
     pub signature_type: u8,
 }
+
+pub struct Media {
+    pub header: Protocol,
+}
+
+impl Media {
+    pub const SUBTYPE_HARD_DRIVE: u8 = 0x01;
+    pub const SUBTYPE_CD_ROM: u8 = 0x02;
```

Comment:
Similar thing here, EDK2 uses `CDROM` as single word. `CD_ROM` is not used once.

Reply:
Amended as requested in latest push.